bazel/api-gen/extraction/test/BUILD.bazel

```diff
@@ -4,8 +4,14 @@ package(default_visibility = ["//bazel/api-gen:__subpackages__"])
 
 extract_api_to_json(
     name = "test",
-    srcs = ["fake-source.ts"],
+    srcs = [
+        "fake-source.ts",
+        "//bazel/api-gen/extraction/test/dummy-entry-point:dummy_package",
+    ],
     entry_point = "fake-source.ts",
+    import_map = {
+        "//bazel/api-gen/extraction/test/dummy-entry-point:index.ts": "@angular/dummy-package",
```

Comment:
How do we deal with secondary entry-points? e.g. if something imports `@angular/dummy-package/testing`? sounds like we'd need to wire up all individually. Usually we'd have a path mapping like `@angular/bla/*` for such then? do we want to automatically add that? (unless there is an explicit mapping obviously)

Reply:
The long-term solution is to just derive these based on the aspect. Then we'd have explicit sub-entries for all secondary/tertiary entry-points as well